typescript/web/next-sitemap.js

```diff
@@ -1,15 +1,5 @@
-const { VERCEL_URL } = process.env;
-
-if (!VERCEL_URL) {
-  throw new Error("Missing required environment variable VERCEL_URL");
-}
-
-const SITE_URL = VERCEL_URL.includes("://")
-  ? VERCEL_URL
-  : `https://${VERCEL_URL}`;
-
```

Comment:
@Jordanlelay Why have you removed those checks?

Reply:
Because I won't be using them as I use a dedicated variable

Reply:
The lines 3 to 5 are still useful.
But you're right for the lines 7 to 9 👍

Reply:
I added it back

Reply:
Yes, but you need to replace VERCEL_URL by LABELFLOW_URL in the check now 😉

Reply:
🤦  fixed